mongomock/aggregate.py

```diff
@@ -249,6 +249,14 @@ def parse_many(self, values):
                 if self._ignore_missing_keys:
                     yield None
 
+    def parse_to_bool(self, expression):
```

Comment:
Please prefix your function name with `_` as it's supposed to be private.

Reply:
Done, thanks!